content/en/api/usage/usage_multi_org.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Get usage across your multi-org account
 type: apicontent
-order: 35.8
+order: 35.94
```

Comment:
nit/follow-up change: this should be updated as well to include the many other new & updated products made currently available [this API https://docs-staging.datadoghq.com/julian/usage-api-updates/api/?lang=bash#get-usage-across-your-multi-org-account]

Sample payload for org 2:
```
{
  "container_avg_sum": 17659,
  "last_updated": "2020-03-08T23",
  "alibaba_host_top99p_sum": 11,
  "aws_lambda_func_count": 85,
  "indexed_events_count_agg_sum": 103495049365,
  "infra_hours_agg_sum": 6168061,
  "netflow_indexed_events_count_agg_sum": 17148394531,
  "fargate_tasks_count_avg_sum": 2,
  "apm_host_top99p_sum": 20666,
  "usage": [
    {
      "custom_output_ts_avg": 110941613,
      "trace_search_indexed_events_count_sum": 36439530002,
      "custom_ts_avg": 329801673,
      "apm_host_top99p": 20666,
      "fargate_tasks_count_avg": 2,
      "infra_hours_sum": 6168061,
      "netflow_indexed_events_count_sum": 17148394531,
      "synthetics_browser_check_calls_count_sum": 770028,
      "aws_lambda_func_count": 85,
      "indexed_events_count_sum": 103495049365,
      "gcp_host_top99p": 58,
      "azure_host_top99p": 633,
      "synthetics_check_calls_count_sum": 13504537,
      "container_avg": 17659,
      "agent_host_top99p": 32773,
      "date": "2020-03",
      "npm_host_top99p": 20863,
      "infra_host_top99p": 33580,
      "ingested_events_bytes_sum": 360396895916192,
      "aws_host_top99p": 395,
      "alibaba_host_top99p": 11,
      "custom_input_ts_avg": 1306933
    }
  ],
  "custom_ts_sum": 329801673,
  "synthetics_check_calls_count_agg_sum": 13504537,
  "custom_output_ts_sum": 110941613,
  "end_date": "2020-03",
  "azure_host_top99p_sum": 633,
  "trace_search_indexed_events_count_agg_sum": 36439530002,
  "npm_host_top99p_sum": 20863,
  "ingested_events_bytes_agg_sum": 360396895916192,
  "synthetics_browser_check_calls_count_agg_sum": 770028,
  "agent_host_top99p_sum": 32773,
  "start_date": "2020-03",
  "aws_host_top99p_sum": 395,
  "gcp_host_top99p_sum": 58,
  "infra_host_top99p_sum": 33580,
  "custom_input_ts_sum": 1306933
}```